src/js/components/LogDetails/UidPanel.tsx

```diff
@@ -1,45 +1,26 @@
 import {useSelector} from "react-redux"
 import React from "react"
 
-import {XUidTimeline} from "../UidTimeline"
 import {reactElementProps} from "../../test/integration"
-import {toFront} from "../../lib/Array"
 import InlineTableLoading from "../InlineTableLoading"
 import LogDetails from "../../state/LogDetails"
 import PanelHeading from "./PanelHeading"
 import {zng} from "zealot"
+import ZeekCorrelation from "../../correlation/components/ZeekCorrelation"
 
 export default function UidPanel({log}: {log: zng.Record}) {
   const status = useSelector(LogDetails.getUidStatus)
   const logs = useSelector(LogDetails.getUidLogs)
-
   return (
     <div
       className="correlated-logs-panel detail-panel"
       {...reactElementProps("correlationPanel")}
     >
-      <PanelHeading status={status}>Uid Correlation</PanelHeading>
-      {logs.length === 0 && (
+      <PanelHeading status={status}>Correlation</PanelHeading>
+      {logs.length === 0 && status !== "SUCCESS" && (
```

Comment:
Sometimes if the search returns no logs, it would show to loading screen forever. Checking that the status is not success helps avoid that.